File: chapters/iw/c/fizzbuzz.c
#include <stdio.h>

void fizzbuzz(int n)
{
     if (n % 3 == 0 && n % 5 == 0)
     {
          printf("Fizz,Buzz\n");
     }
     else if (n % 3 == 0)
     {
          printf("Fizz\n");
     }
     else if (n % 5 == 0)
     {
          printf("Buzz\n");
     }
}

int main(void)
{
     printf("Please input the number.\n");

     int i;
     scanf("%d", &i);

     fizzbuzz(i);

     return 0;
}

File: chapters/iw/src/fizzbuzz.js
const fizzBuzz = (n) => {
    switch (true) {
        case n % 15 == 0:
            console.log('fizzbuzz');
            break;
        case n % 3 == 0:
            console.log('fizz');
            break;
        case n % 5 == 0:
            console.log('buzz');
            break;
    }
}

fizzBuzz(30);
